### PR TITLE
tests sql, read tabledata

### PR DESCRIPTION
**Pull request by MegalexRex**
sqldb
added FindTable
added GetRowData
added getTableStructure 

sql test
added test for many possible conditions

### CODE REVIEW

File: vent/sqldb/sqldb.go [hunk 1]
```diff
@@ -155,49 +360,18 @@ func (db *SQLDB) createTable(table Table) error {
 	return err
 }
 
-// CreateTable creates a new table in the default schema
+// createTable creates a new table in the default schema
 func (db *SQLDB) alterTable(table Table) error {
-	//	"col_1";	"integer";					"int4";			0;	"NO"
```
Comment (hiturria) [inline vent/sqldb/sqldb.go]: in comments should be -> alterTable modify a table structure in the default schema

File: vent/sqldb/sqldb.go [hunk 2]
```diff
@@ -57,6 +59,214 @@ func NewSQLDB(dbURL string, schema string) (*SQLDB, error) {
...
+			}
+		} else {
+			return err
+		}
+	}
+}
+
+// GetRowData Return Table Structure and data of one row
+func (db *SQLDB) GetRowData(tableName string, ids []string) (Table, error) {
+	//Actual table structure in PGSQL
```
Comment (hiturria) [inline vent/sqldb/sqldb.go]: comment should be -> GetRowData returns the table structure and one row of data

File: vent/sqldb/sqldb_test.go [hunk 3]
```diff
@@ -9,7 +9,112 @@ import (
...
+			Columns:   columnsDef,
+		}
+
+		err := db.UpsertRow(tableDef)
+		require.NoError(t, err)
+	})
+
+	//insets a row with aditional fields
+	t.Run("successfully inserts row=3, with extra filds table is mdifiyed", func(t *testing.T) {
 		db, closeDB := test.NewTestDB(t)
```
[the remark this answers is not in the document]
Comment (MegalexRex) [reply vent/sqldb/sqldb_test.go]: ready

File: vent/sqldb/sqldb_test.go [hunk 4]
```diff
@@ -9,7 +9,112 @@ import (
...
+		db, closeDB := test.NewTestDB(t)
+		defer closeDB()
+
+		err := db.DestroySchema()
+		require.NoError(t, err)
+	})
+
+	//first run create schema and table(test_table) and make insert
+	t.Run("successfully creates default schemaa, a table and inserts row=1", func(t *testing.T) {
+		db, closeDB := test.NewTestDB(t)
```
[the remark this answers is not in the document]
Comment (MegalexRex) [reply vent/sqldb/sqldb_test.go]: ready

**Pull request merged**